contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -6859,6 +6859,7 @@ exec_eval_datum(PLtsql_execstate *estate,
 				*typeid = tbl->tbltypeid;
 				*typetypmod = -1;
 				*value = CStringGetDatum(tbl->tblname);
+				*isnull = false;
```

Comment:
Please revive the previous comment as well (https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1267/files) if it is still valid.